packages/java-edition/src/mcfunction/tree/patch.ts

```diff
@@ -677,58 +770,99 @@ const AdvancementTargets: PartialTreeNode = Object.freeze({
 	},
 })
 
-const DataSource = Object.freeze({
-	children: {
-		storage: {
-			children: {
-				source: {
-					properties: {
-						category: 'storage',
+function getDataPatch(
+	key: 'source' | 'target',
+	childKey: 'nbt' | 'path' | 'sourcePath' | 'targetPath' | 'value',
+	{ accessType = SymbolAccessType.Read, children }: {
+		accessType?: SymbolAccessType | undefined
```

Comment:
This `accessType` doesn't seem to be used in the nbt properties. Is that intended?

Reply:
I realize there are different access type information we could include and have added comments to clarify.